src/styles/theme.js

```diff
@@ -1,4 +1,14 @@
-export const PRIMARY = '#E91E63';
```

Comment:
Move this file to `src/theme/theme.js` and everything (theme, colors) needs to be further exported from `src/theme/index.js`